portfolio/src/main/webapp/hobbies.html

```diff
@@ -9,6 +9,125 @@
   <body>
     <div id="content">
       <h1>My Hobbies</h1>
+      <h3>Here is a list of some of my many hobbies (in no particular order):
```

Comment:
Can you keep the h3 tag formatting consistent? E.g. for the h3 tag in line 12, you're adding the closing tag in line 13. Whereas in line 25, the opening and closing tag are in the same line.  It's up to you to pick the style you prefer, as long as it is consistent.

Reply:
Should be fixed!